pipelines/build/prTester/README.md

```diff
@@ -0,0 +1,127 @@
+# PR Testers
+
+openjdk-build is an open source project, therefore, we need to ensure the the code that is being deployed to our master branch doesn't break any existing code and actually works as expected. To achieve this level of testing, we use two jobs to compile the code and run demo pipelines in a controlled sandbox environment (colloquially known as "the pr tester").
+
+## When they're used
+
+Every new pull request to this repository should have the pr tester run on it at least once to verify the changes don't break anything significant (documentation changes being excluded from this rule). The two jobs will appear as as [GitHub Status Checks](https://docs.github.com/en/github/administering-a-repository/about-required-status-checks) at the bottom on the PR being tested:
+![Image of PR Tester Checks](../../../docs/images/pr_tester_checks.png)
+
+## How they're used
+
+The pr testers have their own admin and white lists attached to them on Jenkins.
+If you are on the whitelist or admin list, the pr tester will run against your pr whenever you make one and it will also allow you access to various commands you can run on your own PR or on someone else's:
+
+### `run tests`
+
+- Executes new [openjdk-build-compile-groovy](#openjdk-build-compile-groovy) and [openjdk-build-pr-tester](#openjdk-build-pr-tester) jobs against this PR. These jobs will populate the GitHub status checks field as described above. Please be patient as the [tester does not run concurrently yet](https://github.com/AdoptOpenJDK/openjdk-build/issues/2053) so it may take some time to execute the jobs if there is a long job queue. You can track the progress of it in [Jenkins](https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/) OR look at the status check message:
+
+  - Example of a pr that is in the queue:
+  ![Image of queued tester](../../../docs/images/pr_tester_queued.png)
+  - One that is at the front of the queue and currently being tested:
+  ![Image of building tester](../../../docs/images/pr_tester_building.png)
+
+- When the tester is done, it will return a response comment to the pr with feedback on the testing similar to the following:
+![Image of test result](../../../docs/images/pr_tester_result.png)
+
+- The message will vary depending on the result of the test. Please remember however, that failed tests may be due to existing problems in the nightly builds. If you're unsure if the tests failed because of your changes or not, check our [issue board](https://github.com/AdoptOpenJDK/openjdk-build/issues) and our [triage doc](https://docs.google.com/document/d/1vcZgHJeR8rW8U8OD23Uob7A1dbLrtkURZUkinUp7f_w/edit?usp=sharing) for the existing error
+
+  - 🟢**SUCCESS** 🟢 All the tests and jobs passed, congratulations!
+  - 🟠**FAILURE** 🟠 Some of the tests or jobs failed OR the job was aborted. Check the link in the field at the bottom of the PR for the job link to see exactly where it went wrong.
```

Comment:
What conditions can lead to an abortion of the job? Might be good to explain that.